jobfunnel/glassdoor.py

```diff
@@ -285,7 +284,7 @@ def scrape(self):
             self.scrape_data[str(job['id'])] = job
 
         # Do not change the order of the next three statements if you want date_filter to work
-
+        
```

Comment:
Just cosmetics, but there are four spaces here.

Reply:
Thanks so much for all of your feedback and thoughts on future changes! I'll be sure to implement this change along with Paul's request later today.

Reply:
Has been fixed on 45d3c40 :)